build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
This shebang brings greater portability between OS and is not too strict as a POSIX shell. I think it would be good to have the same shebangs across scripts in the project. Because bash shebang is used in the ```test_commands.sh``` script.

Reply:
Agree, although I thought of the `env` thing. Anyway, it turned out that it's not a big deal.